source/PlayerInfoPanel.cpp

```diff
@@ -505,8 +506,18 @@ void PlayerInfoPanel::DrawPlayer(const Rectangle &bounds)
 		table.Advance();
 		table.DrawGap(5);
 		
-		table.DrawTruncatedPair(combatRating, dim,
-			"(" + to_string(combatLevel) + ")", dim, Truncate::MIDDLE, false);
+		table.DrawTruncatedPair("rank:", dim,
+			to_string(combatLevel), dim, Truncate::MIDDLE, false);
+		table.DrawTruncatedPair("title:", dim,
+			combatRating, dim, Truncate::MIDDLE, false);
+		table.DrawTruncatedPair("experience:", dim,
+			Format::Number(combatExperience), dim, Truncate::MIDDLE, false);
+		if(combatRating == GameData::Rating("combat", combatLevel + 1))
+			table.DrawTruncatedPair("    for next rank:", dim,
+				"MAX", dim, Truncate::MIDDLE, false);
+		else
+			table.DrawTruncatedPair("    for next rank:", dim,
+				Format::Number(ceil(exp(combatLevel + 1))), dim, Truncate::MIDDLE, false);
```

Comment:
Seems like it'd be better to trim this down to the following:
```suggestion
		bool maxRank = (combatRating == GameData::Rating("combat", combatLevel + 1));
		table.DrawTruncatedPair("    for next rank:", dim,
				maxRank ? "MAX" : Format::Number(ceil(exp(combatLevel + 1))),
				dim, Truncate::MIDDLE, false);
```

Reply:
Some day I'll internalize ternary operators.